finetuning/generalists/training/electron_microscopy/mito_nuc/obtain_mito_nuc_em_datasets.py

```diff
@@ -0,0 +1,168 @@
+import os
+import numpy as np
+from math import ceil, floor
+
+from elf.io import open_file
+
+from torch_em import get_data_loader
+from torch_em.transform.label import PerObjectDistanceTransform
+from torch_em.data import ConcatDataset, MinInstanceSampler, datasets
+
+from micro_sam.training import identity
+
+
+class ResizeRawTrafo:
+    def __init__(self, desired_shape, do_rescaling=True, padding="constant"):
+        self.desired_shape = desired_shape
+        self.padding = padding
+        self.do_rescaling = do_rescaling
```

Comment:
This is not actually used. Is that intended? Should it be removed?

Reply:
Yes, this isn't necessary. I will remove it. Thanks!